cmd/system-probe/config/config.go

```diff
@@ -170,8 +171,9 @@ func load(configPath string) (*Config, error) {
 		LogLevel:  cfg.GetString(key(spNS, "log_level")),
 		DebugPort: cfg.GetInt(key(spNS, "debug_port")),
 
-		StatsdHost: aconfig.GetBindHost(),
-		StatsdPort: cfg.GetInt("dogstatsd_port"),
+		StatsdHost:                   aconfig.GetBindHost(),
+		StatsdPort:                   cfg.GetInt("dogstatsd_port"),
+		DefaultSystemProbeExpVarPort: cfg.GetString(key(spNS, "expvar_port")),
```

Comment:
Since we're reading this value from a config file, IMO having `Default` in this variable name is misleading as it may or may not actually have the default value `5020`. WDYT about just naming it `ExpVarPort` instead?

Reply:
Also - should it be a `string`? `int` seems more appropriate

Reply:
Agreed with the `Default` point. Changing it to `SystemProbeExpvarPort` so it remain explicit.

I kept it as string since that was how it was used in the code. But on second though int is the more appropriate datatype.